style-about.css

```diff
@@ -0,0 +1,697 @@
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding-bottom: 4vw;
+}
+
+.partner h2 {
+  color: var(--main-color-gray);
+  font-size: 5vw;
+  padding-top: 6vw;
+  padding-bottom: 5px;
+}
+
+.partner .line {
+  border-top: 1px solid var(--main-color-orange);
+  width: 8vw;
+  margin-top: 1vw;
+  margin-bottom: 6vw;
+}
+
+.partner .container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.partner-all {
+  display: none;
+}
+
+.partner .container img {
+  margin: 0;
+  padding: 0;
+  width: 22vw;
+}
+
+/* OUR PARTNERS */
+
+/* LICENSE */
+.license {
+  padding: 3vw 0;
+  background-color: white;
+}
+
+.license .container {
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-right: 0;
+}
+
+.license .container .img-for-black-bg {
+  display: none;
+}
+
+.license .container img {
+  width: 50%;
+  padding: 20px;
+  align-items: center;
+}
+
+.license-info p {
+  font-size: x-small;
+}
+
+.license .container .korian-lan {
+  display: none;
+}
+
+@media (width > 755px) {
+  .menu {
+    display: none;
+  }
+
+  .nav2 .cancle {
+    display: none;
+  }
+
+  .nav2[item-visibility="false"] {
+    transform: translateX(0%);
+  }
+
+  .softnav {
+    display: block;
+    position: fixed;
+    top: 113px;
+    right: 10vw;
+    padding-right: 5px;
+  }
+
+  .softnav a {
+    padding: 0 7px;
+    text-decoration: none;
+    color: #454545;
+  }
+
+  .softnav a i {
+    padding: 0 7px;
+  }
+
+  .header {
+    background-color: #fff;
+    border-top-right-radius: 30px;
+    width: 100%;
+    position: fixed;
+  }
+
+  .nav2 .logo {
+    margin: 0;
+  }
+
+  .logo {
+    margin: 5px 0 2px 0;
+  }
+
+  .logo:hover {
+    transform: scale(1.1);
+  }
+
+  .nav1,
+  .nav2 {
+    position: static;
+    display: flex;
+  }
+
+  .container {
+    padding: 0 10vw;
+  }
+
+  .skew-box1,
+  .skew-box2 {
+    height: 30px;
+  }
+
+  .skew-box1 {
+    background-color: rgb(65, 65, 65);
+    position: relative;
+    left: -1%;
+    width: 51.5vw;
+    z-index: 2;
+    transform: skew(30deg);
+  }
+
+  .skew-box2 {
+    background-color: rgba(0, 0, 0, 0.4);
+    position: absolute;
+    right: 0;
+    width: 50.5vw;
+    z-index: 1;
+  }
+
+  .topnav {
+    position: absolute;
+    width: 90vw;
+    height: 30px;
+    margin: 0 auto;
+    right: 10vw;
+    z-index: 3;
+  }
+
+  .nav1 ul,
+  .nav2 ul {
+    margin: 0;
+    padding: 0;
+    display: flex;
+  }
+
+  .nav1 ul {
+    align-items: center;
+    justify-content: flex-end;
+  }
+
+  .nav1 ul li,
+  .nav2 ul li {
+    list-style: none;
+    padding: 3px 15px 0 12px;
+  }
+
+  .nav1 a,
+  .nav2 a {
+    text-decoration: none;
+    font-weight: 700;
+    color: white;
+  }
+
+  .nav2 a {
+    color: rgb(17, 16, 16);
+  }
+
+  /*
+  .nav2 a:hover {
+    text-decoration: none;
+    font-weight: 700;
+    color: var(--main-color-orange);
+  } */
```

Comment:
- [Optional] Please consider removing `commented codes` as it will make your codebase messy, Remember if you ever like to see the codes you have written once you can use your commit history.